src/server/scripts/Outland/CoilfangReservoir/SerpentShrine/boss_lady_vashj.cpp

```diff
@@ -191,6 +192,7 @@ class boss_lady_vashj : public CreatureScript
         uint32 CoilfangStriderTimer;
         uint32 SummonSporebatTimer;
         uint32 SummonSporebatStaticTimer;
+        uint32 uint32 ParalyzecCheckTimer;
```

Comment:
uint32 uint32 ??
also please make sure to initialized this field just like all other fields are initialized